Wire.h fails

Running version 2.0.1.

Reproduce :
Make an empty sketch and just add  at the top : #include "Wire.h"
Try to compile and it fails with:
In file included from /home/paul/Arduino/sketch_oct07a/sketch_oct07a.ino:1:
/home/paul/.arduino15/packages/SparkFun/hardware/apollo3/2.0.1/libraries/Wire/src/Wire.h:11:10: fatal error: Print.h: No such file or directory
 #include "Print.h"

Root cause
Print.h is now in mbed-bridge/core-api/api directory.

Solution
Either change the line to #include "mbed-bridge/core-api/api/Print.h" or just remove #include "Print.h" from Wire.h

regards,
Paul

I have just tried again. I am using IDE 1.8.13. 
I have removed and reinstalled the 2.0.1 library to make sure I had not changed anything and make a clean start.
Empty sketch with only #include <Wire.h> in the top. Nothing else..no code. Try to compile give the error  : 

In file included from /home/paul/Arduino/sketch_oct12a/sketch_oct12a.ino:1:
/home/paul/.arduino15/packages/SparkFun/hardware/apollo3/2.0.1/libraries/Wire/src/Wire.h:10:10: fatal error: Print.h: No such file or directory
 #include "Print.h"
          ^~~~~~~~~
Indeed it is not necessary to include the Print.h in this file, as it all compiles without any problem is commented out.

--- a/libraries/Wire/src/Wire.h
+++ b/libraries/Wire/src/Wire.h
@@ -7,7 +7,6 @@
 #define _APOLLO3_LIBRARIES_WIRE_H_
 
 #include "Arduino.h"
-#include "Print.h"
 #include "drivers/I2C.h"
 #include "drivers/I2CSlave.h"
 #include "rtos.h"

